paddle/fluid/platform/device_tracer.cc

```diff
@@ -30,9 +30,6 @@ limitations under the License. */
 namespace paddle {
 namespace platform {
 namespace {
-// Current thread's id. Note, we don't distinguish nested threads
-// for now.
-thread_local int cur_thread_id = 0;
```

Comment:
?

Reply:
No. I'm using another way to track threads. See change in profiler.cc